sematic/ui/packages/main/src/components/BidirectionalLogView.tsx

```diff
@@ -0,0 +1,161 @@
+import { useCallback, useEffect, useState } from "react";
```

Comment:
Initially only the "jump to end" button appears. After clicking it, it disappears, and "jump to beginning" appears. The reverse is also true.

After scrolling a bit, and not viewing either the end or the beginning of the logs, I would like to be able to seek to the end in both directions, not only in the direction I didn't seek in most recently.

So is it possible to always display both "jump" buttons if not viewing one of the ends, and have them always seek + also fetch if it is the case?